src/lib/entity-state.ts

```diff
@@ -159,13 +166,14 @@ export abstract class EntityState<T> {
   /**
    * Returns a selector for paginated entities, sorted by insertion order
    */
-  static paginatedEntities(size: number, page: number): StateSelector<any[]> {
+  static get paginatedEntities(): StateSelector<any[]> {
     // tslint:disable-line:member-ordering
     const that = this;
     return state => {
       const subState = elvis(state, that.staticStorePath) as EntityStateModel<any>;
-      return subState.ids
-        .slice(page * size, (page + 1) * size)
+      const { ids, pageIndex, pageSize } = subState;
+      return ids
+        .slice(pageIndex * pageSize, (pageIndex + 1) * pageSize)
```

Comment:
I think it'd be more readable to name this key pageNumber, that starts from 1.

Reply:
I think pageIndex is preferable. angular/material's paginator also uses `pageIndex` starting at 0. The suffix "index" should clarify things. I will add this in the documentation.